packages/drivers/file-driver/src/fileDocumentStorageService.ts

```diff
@@ -53,9 +53,9 @@ export class FluidFetchReader extends ReadDocumentStorageServiceBase implements
                 return null;
             }
             rootTree = true;
-            filename = `${this.path}/${this.versionName}/decoded/tree.json`;
+            filename = `${this.path}/${this.versionName}/tree.json`;
```

Comment:
So now we are reading from non decoded folder, where is the conversion happening in order to read this correctly? because previously we were reading the decoded tree.

Reply:
Tree is always a string.  the only different between the two is whether we format it or not.  See `saveSnapshot` in `fluidFetchSnapshot.ts`

Reply:
Feels like I added it here: https://github.com/microsoft/FluidFramework/pull/5227